kubernetes/ftest-kubernetes-logs/pom.xml

```diff
@@ -25,6 +25,11 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.fabric8</groupId>
```

Comment:
Just some minor thing. Can we add this dependency under kubernetes/pom.xml instead of adding to each sub modules ?

Reply:
@jimma the dependency is needed by 5 out of 9 child projects, and we don't really want for others that don't need it to inherit it, e.g.: the `istio` module should not depend on this artifact as they don't need it. The dependency management is already defined by the root POM, so in the end I don't think we should bring it up.
Do you agree?